NitroxPatcher/Patches/Dynamic/FreezeTime_Set_Patch.cs

```diff
@@ -6,15 +6,15 @@
 namespace NitroxPatcher.Patches.Dynamic;
 
 /// <summary>
-/// Because we're in multiplayer mode, we generally don't want the game to freeze
+/// Because we're in multiplayer mode, we generally don't want the game to freeze.
+/// Thus we prevent any FreezeTime that is not for quit screen or (initial sync) wait screen.
```

Comment:
For what is `FreezeTime.Id.Quit` used in base sn? If it's for the esc menu isn't that against our intend?

Reply:
Quit is only used by IngameMenu.QuitGameAsync (which means you're already leaving the world) (quit screen means the saving screen and then when going back to the menu).
Also we have modals like, KickedModal, ServerStoppedModal and LostConnectionModal (when we open those, the client is already disconnected from the server) which use Quit because it's the only FreezeTime id other than WaitScreen that is relevant for this purpose.

I believe Esc menu's freeze id is IngameMenu